Remotion/Validation/Core/Validators/MinimumLengthValidator.cs

```diff
@@ -15,18 +15,78 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
 using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
 using JetBrains.Annotations;
+using Remotion.FunctionalProgramming;
+using Remotion.Utilities;
 using Remotion.Validation.Implementation;
+using Remotion.Validation.Results;
 
 namespace Remotion.Validation.Validators
 {
-  public class MinimumLengthValidator : LengthValidator
+  public class MinimumLengthValidator : ILengthValidator
   {
+    public int Min { get; }
+
+    public int? Max { get; }
```

Comment:
Only Min is configurable. Max is an explicit interface implementation. Update the code in the class as needed.